diff -rup malayalam-fonts-04/malayalam-fonts.conf malayalam-fonts-04_mod/malayalam-fonts.conf
--- malayalam-fonts-04/malayalam-fonts.conf	2008-06-30 00:46:25.000000000 +0530
+++ malayalam-fonts-04_mod/malayalam-fonts.conf	2010-02-25 20:00:06.000000000 +0530
@@ -2,29 +2,6 @@
 <!DOCTYPE fontconfig SYSTEM "fonts.dtd">
 <fontconfig>
 <!-- Malayalam (ml) -->
-<match target="font">
-        <test name="lang" compare="contains">
-                <string>ml</string>
-        </test>
-        <alias>
-                <family>sans-serif</family>
-                <prefer>
-                        <family>Meera</family> 
-                </prefer>
-	</alias>
-</match>
-
-<match target="font">
-        <test name="lang" compare="contains">
-                <string>ml</string>
-        </test>
-        <alias>
-                <family>serif</family>
-                <prefer>
-                        <family>Rachana</family>
-                </prefer>
-        </alias>
-</match>
 
 <!-- multiply the matrix of Meera font for solving size mismatch with Rachana-->
 <match target="font">
